offsets:
  servo0: 0.0
  servo1: 0.0
  servo2: 0.0
  servo3: 0.0
  servo4: 0.0
  servo5: 0.0
  servo6: 0.0
  servo7: 0.0
  servo8: 0.0
  servo9: 0.0
  servo10: 0.0
  servo11: 0.0
